### ISSUE TITLE
[v5] Allow links with custom protocol in markdown for log entries

### ISSUE THREAD

**Issue by nvborisenko** [reportportal/reportportal#1237]
Links in markdown with custom protocol are not clickable.

# To Reproduce

Send  log message to RP:

```
any **bold** text
[my_link](vscode://abc.com)
```

This log message is shown as:
![image](https://user-images.githubusercontent.com/22616990/100915630-ae381c00-34e5-11eb-9bb3-1a65158e4902.png)

Visually the link is rendered correctly, but user cannot click on this link.
![image](https://user-images.githubusercontent.com/22616990/100915804-ea6b7c80-34e5-11eb-8fe0-7854894ebd57.png)

# Additional info
If we send a link as
```
any **bold** text
[my_link](https://abc.com)
```
then renderer works perfectly.

# Versions
 - OS: Win10
 - Browser: Chrome 87
 - RP Version: Service UI 5.2.3

**Comment by evjlobanova** [comment]
@svettsova Valya could you please add a bug to Jira

**Comment by svettsova** [comment]
Defect is added to jira

**Comment by Kozovskaya** [comment]
fixed in version 5.4
